cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server.go

```diff
@@ -72,473 +40,46 @@ type Server struct {
 	// clientGetter is a field so that it can be switched in tests for
 	// a fake client. NewServer() below sets this automatically with the
 	// non-test implementation.
-	clientGetter clientGetter
+	clientGetter             clientGetter
+	globalPackagingNamespace string
+	globalPackagingCluster   string
```

Comment:
These surprised me? I'm sure i'll see below why they were needed.

Reply:
I'm adding these lines for defaulting to the Kubeapps cluster if no cluster is passed in the request and for entering the `kapp-controller-packaging-global` namespace if required at some point.
At this very moment, I don't know either if it's gonna be 100% required, but I added for similarity as in other plugins.